cross/toolchain.cmake

```diff
@@ -35,15 +35,6 @@ if(DEFINED ENV{TOOLCHAIN})
   set(TOOLCHAIN $ENV{TOOLCHAIN})
 endif()
 
-# This gets called with CLR_CMAKE_COMPILER set on the first invocation
-# but the cmake variable is not getting populated into other calls.
-# Use environment variable to keep CLR_CMAKE_COMPILER around.
-if (NOT DEFINED CLR_CMAKE_COMPILER)
```

Comment:
See this comment here. You need to bring the CLR_CMAKE_COMPILER setting back to life.

# This gets called with CLR_CMAKE_COMPILER set on the first invocation
# but the cmake variable is not getting populated into other calls.
# Use environment variable to keep CLR_CMAKE_COMPILER around.